src/main/java/com/fcgl/madrid/forum/model/Comment.java

```diff
@@ -90,4 +92,12 @@ public Post getPost() {
     public void setPost(Post post) {
         this.post = post;
     }
+
+    public int getUserId() {
+        return userId;
+    }
+
+    public void setUserId(int userId) {
```

Comment:
Think this should be removed as who made the comment should be able to change.

Reply:
Maybe I'll make it private? I think this is required by Spring. Could be wrong. Will try without it

Reply:
Okay, removed any setters that we don't need